glooshot/docs/_index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Gloo Shot
+weight: 99
+---
+
+#### Service Mesh Chaos Engineering
+
+Gloo Shot is a chaos engineering framework for service meshes.
+
+
+## Using Gloo Shot
+- **Harden your mesh**: Gloo Shot allows you to test failure modes before they occur in production.
```

Comment:
So I guess the focus is on application failure modes (and could be caused by supporting infra).. but let's change this to something like "Harden your applications" instead of the mesh specifically, WDYT?

Reply:
Agreed, that's a better way to put it